docs/SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you find a security vulnerability in the Cirrus CI platform (the backend, web interface, etc.), please **follow the steps below**.
+
+1. **Do NOT comment about the vulnerability publicly.** *Doing so would be in violation of our Terms of Service, [and may have consequences](https://cirrus-ci.org/legal/terms/#cancellation-and-termination)!*
```

Comment:
I think we can skip 

> *Doing so would be in violation of our Terms of Service, [and may have consequences](https://cirrus-ci.org/legal/terms/#cancellation-and-termination)!*

IMO looks scary 😅

Reply:
Thats the point, we should show that public disclosure is a big no no

Reply:
I think people who will read it will know it. People who disclose such issues don't read policies. 🤷‍♂

Reply:
True

Reply:
Fixed now